VideoLocker/src/androidTest/java/org/edx/mobile/test/http/SegmentTests.java

```diff
@@ -462,8 +469,8 @@ public void testtrackEnrollClicked() throws Exception {
         Properties props = segment.trackEnrollClicked(courseId, email_opt_in);
 
         // verify that the identity method was called
-        Mockito.verify(tracker).track(Mockito.eq(ISegment.Keys.SIGN_UP),
-                (Properties) Mockito.any());
+//        Mockito.verify(tracker).track(Mockito.eq(ISegment.Keys.SIGN_UP),
+//                (Properties) Mockito.any());
```

Comment:
Why is this code commented out?  I prefer not merging commented out code.  Either delete it if we no longer need it.  Or at least have a comment to explain why it's commented out.